src/tools/muon.ts

```diff
@@ -87,24 +87,22 @@ export async function format(
   return [new vscode.TextEdit(documentRange, stdout)];
 }
 
-export async function check(): Promise<{ tool: Tool, error: string }> {
+export async function check(): Promise<{ tool?: Tool, error?: string }> {
   const muon_path = extensionConfiguration("muonPath");
-
-  const undef_muon = { path: undefined, version: undefined };
-
   let stdout: string, stderr: string;
 
   try {
-    ({ stdout, stderr } = await exec(muon_path, ["version"]))
-  } catch (exception) {
-    const { error, stdout, stderr }: { error: cp.ExecException, stdout: string, stderr: string } = exception;
+    ({ stdout, stderr } = await exec(muon_path, ["version"]));
```

Comment:
Why is the catch on its own line?

Reply:
No reason, other than no hook is enforcing style 😁